ghost/core/core/server/data/migrations/versions/5.11/2022-08-22-11-03-add-free-member-signup-notif-column-to-users.js

```diff
@@ -0,0 +1,7 @@
+const {createAddColumnMigration} = require('../../utils');
+
+module.exports = createAddColumnMigration('users', 'free_member_signup_notification', {
+    type: 'boolean',
+    nullable: false,
+    defaultTo: false
```

Comment:
Why are we defaulting to false?

Reply:
For existing sites, we want all the notifications to be switched off by default, and we'll notify via changelog if anyone wants to turn them on. So we are defaulting to false on all migrations.

For new sites, we only want the owner to have the notifications on by default during setup, everyone else will by default have them as off.

Reply:
Am following up with @prschulz about this, I think we are going to change it to on by default.

Reply:
Ah ok, in that case, I guess we'll still need to keep it as `false` in this migration, and then add another migration that goes over all `owner`(and `administrator` if we want them) users only and toggles them to `true`.
We do have a check in email logic to ignore non owner/admin users irrespective of their value for this column so all users having `true` value won't affect final email sent, but guess it still makes sense to set the right values for users from start.

Reply:
I really think these should be true by default at the database layer. That migration is just unnecessary extra work.

Reply:
Cool, will update the migrations and default values to true. 👍

Reply:
Note: I'm still discussing with Peter and haven't got absolute confirmation.